packages/wallet/src/redux/state.ts

```diff
@@ -277,8 +288,12 @@ export function checkAndInitialize(
   }
 }
 
-export function checkAndStore(state: SharedData, signedState: SignedState): CheckResult {
-  const result = checkAndStoreChannelStore(state.channelStore, signedState);
+export function checkAndStore(
```

Comment:
Since this method already has access to the `sharedData` it doesn't have to get passed in the `byteCode`. That way protocols don't even have to know about `byteCode` at all.

We could have something like this:
```
export function checkAndStore(state: SharedData, signedState: SignedState): CheckResult {
 const byteCode = sharedData.bytecodeStorage[ss.state.appDefinition];
if (!byteCode){
    throw new Error('Uh oh');
}
const result = checkAndStoreChannelStore(state.channelStore, signedState, bytecode);
....
```